frontend/utilities/endpoints.ts

```diff
@@ -34,11 +34,15 @@ export default {
   LOGIN: `/${API_VERSION}/fleet/login`,
   LOGOUT: `/${API_VERSION}/fleet/logout`,
   MACADMINS: `/${API_VERSION}/fleet/macadmins`,
+  // TODO: Clean up MDM endpoints to be consistent and up to date
   MDM_APPLE: `/${API_VERSION}/fleet/mdm/apple`,
   MDM_APPLE_BM: `/${API_VERSION}/fleet/mdm/apple_bm`,
   MDM_APPLE_BM_KEYS: `/${API_VERSION}/fleet/mdm/apple_bm/keys`,
   MDM_SUMMARY: `/${API_VERSION}/fleet/hosts/summary/mdm`,
+  // Should below 2 endpoints be consistent?
```

Comment:
I don't think they need to be. The above are general MDM operations, and these are specific to a host.